Firmware/ultralcd.cpp

```diff
@@ -4823,7 +4823,7 @@ uint8_t choose_menu_P(const char *header, const char *item, const char *last_ite
         for (uint_least8_t i = 0; i < ordinary_items; i++)
         {
             lcd_set_cursor(2 + item_len, i+1);
-            lcd_print(first + i + 1);
```

Comment:
Note over 100B are saved here, I havent looked at the assemly but maybe I am overlooking something here.

Reply:
Looking at the assembly, it looks like with this change the compiler has decided to inline `lcd_print(unsigned long, int)` here.

Reply:
We should tag `lcd_print` as `__attribute__((noinline))` to prevent future surprises